public/main.js

```diff
@@ -3,6 +3,7 @@ var todoListPlaceholder = document.getElementById("todo-list-placeholder");
 var form = document.getElementById("todo-form");
 var todoTitle = document.getElementById("new-todo");
 var error = document.getElementById("error");
+var todoCount = 0;
```

Comment:
I think this needs a more descriptive name, is it counting the todos or completed todos?